### ISSUE TITLE
didChange does not fire on git pull

### ISSUE THREAD

**Issue by FrederikPM** [jonaskrask/vdm-vscode#150]
If a VDM model is updated by pulling the source from GitHub, the Client doesn't seem to fire the usual "change watched file" events. So the server carries on with the "old" version, which can be confusing(!). If you restart the Client, then of course it's all fine.

This issue looks relevant, though different: https://github.com/microsoft/vscode/issues/72224

**Comment by FrederikPM** [comment]
This is what I have done to try and reproduce the issue:
Add `Test = nat;` to types in alarm.vdmsl in the project `AlarmSL` on PC _A_ and push that to the repository and then pull the changes on PC _B_.
Looking at the LSP log on PC _B_ the event `textDocument/didChange` with the expected content changes is fired when pulling the changes on PC B.

However, this seems to show that it does work?

**Comment by nickbattle** [comment]
Yes, that test should be enough. It's possible that they've recently fixed this(!). It was a while ago when I noticed the problem and I know that there have been VSCode updates since. Or it might be a Linux/Windows/Mac difference (I'm Linux). I'll try again when I get a minute...
